README.md

```diff
@@ -122,7 +122,7 @@ The proxy is also available as a [docker image](Dockerfile):
 ```shell
 $ docker run -p 8081:8080 -e TOKEN_ISSUER=[token_issuer_url] \
   -e PROXY_TO=[fhir_server_url] -e ACCESS_CHECKER=list \
-  gcr.io/second-scion-309318/fhir-proxy:latest
+  us-docker.pkg.dev/fhir-proxy-build/stable/fhir-access-proxy:latest
```

Comment:
Isn't it lovely that these image names keep getting longer and longer?! :-)

Reply:
I know  🙃